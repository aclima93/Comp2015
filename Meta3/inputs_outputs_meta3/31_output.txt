Line 4, col 4: Incompatible type in assignment to integer (got _type_, expected _integer_)
