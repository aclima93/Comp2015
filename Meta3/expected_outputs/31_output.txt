Line 2, col 19: Symbol abc not defined
